### ISSUE TITLE
Floats are rounded when they should not be

### ISSUE THREAD

**Issue by denis-soundcloud** [porras/crul#18]
### Expected behavior

Do not reformat float. Example with`curl`:

```
% curl http://example.com/tracks/123 | jq . | grep bpm
    "bpm": 128.10000610351562,
```

(This uses `jq` to pretty-print the JSON.)
### Actual behavior

`crul` reformats floats. Example:

```
% crul http://example.com/tracks/123 | grep bpm
    "bpm": 128.1,
```

**Comment by asterite** [comment]
I _think_ it's because crul was compiled with a very old crystal version where floats weren't formatted correctly. Compiling with a newer compiler should fix this... except that crul doesn't seem to compile anymore with the latest compiler due to breaking changes (and a required dependency, `completion`, doesn't compile either)

**Comment by porras** [comment]
You were right @asterite, the specific example reproduced by @denis-soundcloud was fixed by #20:

```
$ curl http://localhost:4568/ | jq . | grep float
  "float": 128.10000610351562

$ crul http://localhost:4568/ | grep float
  "float": 128.10000610351562
```

(I'm running a small app that returns that, of course)

Thanks, you two!
